src/models/CreditNote.js

```diff
@@ -68,23 +68,18 @@ const CreditNoteSchema = mongoose.Schema(
     subscription: {
       _id: { type: mongoose.Schema.Types.ObjectId },
       service: {
-        serviceId: {
-          type: mongoose.Schema.Types.ObjectId,
-          required() {
-            return !!this.subscription;
-          },
-        },
+        serviceId: { type: mongoose.Schema.Types.ObjectId },
         nature: {
           type: String,
           enum: SERVICE_NATURES,
           required() {
-            return !!this.subscription;
+            return !!this.subscription.serviceId;
```

Comment:
C'est pas this.subscription.service.serviceId ?

Reply:
Si '-_-